python/requirements.txt

```diff
@@ -19,6 +19,7 @@ aiosignal
 frozenlist
 requests
 virtualenv>=20.0.24, < 20.21.1
+packaging
```

Comment:
If we unpin here, we should also unpin in `setup.py`

Reply:
Oh I see. I'll unpin it and also remove all the `distutils` usage. accordingly